README.md

```diff
@@ -1,8 +1,8 @@
-[![Build Status](CI_BADGE_URL goes here)](REPO_URL goes here)
+[![Test](https://github.com/nimblehq/compass-credo-plugin/actions/workflows/test.yml/badge.svg)](https://github.com/nimblehq/compass-credo-plugin/actions/workflows/test.yml)
 
 ## Introduction
 
-> *App introduction goes here ...*
+The repo contains custom Credo rules to support [Nimble's Elixir conventions](https://nimblehq.co/compass/development/code-conventions/elixir/).
```

Comment:
```suggestion
The repo contains custom [Credo Checks](https://hexdocs.pm/credo/Credo.Check.html) to support [Nimble's Elixir conventions](https://nimblehq.co/compass/development/code-conventions/elixir/).
```

Or whatever the correct syntax is for links 😆

Reply:
Thanks. It was updated here: https://github.com/nimblehq/compass-credo-plugin/pull/23/commits/c05d3b9e48d1aaa2e873cd77ee35973ce2115a4a